UI/static/main.css

```diff
@@ -120,4 +150,9 @@ footer a {
     .container {
         width: 75%;        
     }
+}
+
+/* Handle wide tables */
+.long-x-axis {
+    overflow-x: auto;
 }
```

Comment:
Add new line

Reply:
I will add `\n`. 
Regarding the wide tables on small devices, I will try to implement colapsing tables next time. I'm counting on the grey `highlight` on `hover` will aid the user to keep track of selscted row as they scroll horizontally.